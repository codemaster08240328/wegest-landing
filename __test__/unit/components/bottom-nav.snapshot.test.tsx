import renderer from 'react-test-renderer'
import BottomNav from '@/components/layout/bottom-nav'

jest.mock('next/router', () => ({
  __esMModule: true,
  useRouter: () => ({
    pathname: '/page1',
    route: '/page1',
    push: jest.fn(),
  }),
}))

const routes = [
  { href: '/page1', label: 'page1', icon: 'looks_one' },
  { href: '/page2', label: 'page2', icon: 'looks_two' },
]

describe('<BottomNav> snapshot', () => {
  it('renders correctly', () => {
    expect(
      renderer.create(<BottomNav routes={routes} />).toJSON(),
    ).toMatchSnapshot()
  })
})
