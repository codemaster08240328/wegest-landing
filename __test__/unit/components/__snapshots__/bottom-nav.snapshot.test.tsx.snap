// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BottomNav> snapshot renders correctly 1`] = `
<div
  className="MuiBottomNavigation-root"
>
  <button
    className="MuiButtonBase-root MuiBottomNavigationAction-root Mui-selected"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiBottomNavigationAction-wrapper"
    >
      <span
        aria-hidden={true}
        className="material-icons MuiIcon-root"
      >
        looks_one
      </span>
      <span
        className="MuiBottomNavigationAction-label Mui-selected"
      >
        page1
      </span>
    </span>
  </button>
  <button
    className="MuiButtonBase-root MuiBottomNavigationAction-root"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiBottomNavigationAction-wrapper"
    >
      <span
        aria-hidden={true}
        className="material-icons MuiIcon-root"
      >
        looks_two
      </span>
      <span
        className="MuiBottomNavigationAction-label"
      >
        page2
      </span>
    </span>
  </button>
</div>
`;
